Environment/Environment.pillar

```diff
@@ -294,9 +296,11 @@ of overridden methods (see Figure *OBInheritanceOverriding*).
 By default, the browser presents a list of packages in the leftmost pane.
 However it is possible to switch to a class hierarchy view. Simply select a
 particular class of interest, such as ==ImageMorph== and then click on the
-==hier.== button. You will then see in the left-most pane a class hierarchy
+==Hierarchy== button. You will then see in the left-most pane a class hierarchy
```

Comment:
Minor detail: the hierarchy is displayed in the second pane (the class pane), not the left-most.